src/sphinx_ansible_highlighter/__init__.py

```diff
@@ -1,7 +1,7 @@
 # coding: utf-8
 # Author: Felix Fontein <felix@fontein.de>
-# License: GPLv3+
-# Copyright: Ansible Project, 2020
+# License: BSD-2-Clause
+# Copyright: Felix Fontein <felix@fontein.de>, 2021
```

Comment:
This shouldn't be 2021 but the year you initially wrote this code which seems to be 2018 per https://github.com/ansible/ansible/blame/be9f8c6/docs/docsite/_extensions/pygments_lexer.py#L167-L188.
2021 may optionally be mentioned since you've added changes in 2021 but it's not the start of your copyright.

P.S. I think the legal folks recommended us at Ansible Core not to touch the copyright lines at all. Plus from the explanations, I read elsewhere, updating it to the current year doesn't actually do anything other than imposing the maintenance burden of bumping a number every year. You own a copyright for the thing regardless of whether you've told others about it or not and it starts at the moment of creation.

Reply:
Legal's advice about stating copyright years basically boil down to: someone wanting to use code which is outside of copyright protection under an incompatible license (because the code has fallen into the public domain and therefore can be used irregardless of the author's licensing wishes) cannot trust the year in the comments because that is updated in different ways by different upstreams and those practices don't impact the real end date for copyright protection.  So the year you put here doesn't matter that much.

Legal advises Red Hat associates to use the year they created the software so that they don't have to update the copyright year as they make updates.

Someone who wanted to use the code only after it, in its entirety, has fallen into the public domain would have to base their time period off of when the last copyrightable change was made so 2021 may be more helpful for that person.  Someone who is willing to dig, could find that the bulk of the code was written in 2018 and could conceivably take the old version and use it when that copyright expired instead.  That person might be more interested in the 2018 date.  In either case, the code doesn't fall into the public domain based on the date in the copyright statement; they need to consult when the code they want to use was actually written to figure that out.

With all that background in mind, felix can put 2021 here if he wants.  Since copyrightable changes were made this year, Its no less accurate than writing 2018 in the year.